modules/medium-sev-security-policies.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * operating/default-security-policies.adoc
+:_module-type: REFERENCE
+[id="medium-sev-security-policies_{context}"]
+= Medium severity security policies
+
+////
+In the title of a reference module, include nouns that are used in the body text. For example, "Keyboard shortcuts for ___" or "Command options for ___." This helps readers and search engines find the information quickly.
+
+Be sure to include a line break between the title and the module introduction.
+////
+
+[role="_abstract"]
+The following table lists the default security policies in {product-title} that are of medium severity. The policies are organized by life cycle stage.
+
+.Medium severity security policies
+[options="header"]
+|====
+|Life cycle stage |Name |Description | Status
+
+|Build |Docker CIS 4.4: Ensure images are scanned and rebuilt to include security patches |Alerts when images are not scanned and rebuilt to include security patches. It is important to scan images often to find vulnerabilities, rebuild the images to include security patches, and then instantiate containers for the images. |Disabled
+|Deploy |30-Day Scan Age |Alerts when a deployment has not been scanned in 30 days. |Enabled
+|Deploy |CAP_SYS_ADMIN capability added |Alerts when a deployment includes containers that are escalating with CAP_SYS_ADMIN. |Enabled
+|Deploy |Container using read-write root filesystem |Alerts when a deployment includes containers that have read-write root file systems. |Enabled
+|Deploy |Container with privilege escalation allowed |Alerts when a container might be running with unintended privileges, creating a security risk. This situation can happen when a container process that has more privileges than its parent process allows the container to run with unintended privileges. |Enabled
+|Deploy |Deployments should have at least one ingress Network Policy |Alerts if deployments are missing an ingress Network policy. |Enabled
```

Comment:
Should "Network" be capitalized here? [Ingress](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#ingress) is usually capitalized

Reply:
Good question! Per the Kubernetes glossary, "Network Policy" is capitalized. I'll update the topic to capitalize "ingress" and the second instance of "policy"